cleaners/windows_explorer.xml

```diff
@@ -18,79 +18,127 @@
-    <!-- refresh Windows shell after UserAssist -->
+    <!-- Refresh Windows shell after UserAssist: -->
     <action command="win.shell.change.notify"/>
-    <!-- reference: reference: http://support.microsoft.com/kb/142298 -->
-    <!-- common open dialog -->
+    <!-- Reference: reference: http://support.microsoft.com/kb/142298 -->
+    <!-- Common open dialog: -->
     <action command="winreg" path="HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\ComDlg32\LastVisitedMRU"/>
-    <!--  Windows Vista/7 -->
+    <!-- Windows Vista/7: -->
     <action command="winreg" path="HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\ComDlg32\LastVisitedPidlMRU"/>
-    <!--  common save as dialog -->
+    <!-- Common save as dialog: -->
     <action command="winreg" path="HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\ComDlg32\OpenSaveMRU"/>
-    <!--  Windows Vista/7 -->
+    <!-- Windows Vista/7: -->
     <action command="winreg" path="HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\ComDlg32\OpenSavePidlMRU"/>
-    <!--  find files command -->
+    <!-- Find files command: -->
     <action command="winreg" path="HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\Doc Find Spec MRU"/>
-    <!--  find Computer command -->
+    <!-- Find computer command: -->
     <action command="winreg" path="HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\FindComputerMRU"/>
-    <!--  map network drives -->
+    <!-- Map network drives: -->
     <action command="winreg" path="HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\Map Network Drive MRU"/>
-    <!--  printer ports -->
+    <!-- Printer ports: -->
     <action command="winreg" path="HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\PrnPortsMRU"/>
+    <!-- More MRUs: -->
+    <!--
+    If CCleaner deletes HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\ComDlg32 complete, we can be generous! ;-) Tobias.
+     -> Beginning... -->
+    <action command="winreg" path="HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\ComDlg32\CIDSizeMRU"/>
+    <action command="winreg" path="HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\ComDlg32\FirstFolder"/>
+    <action command="winreg" path="HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\ComDlg32\LastVisitedPidlMRULegacy"/>
+    <!-- End! -->
+    <action command="winreg" path="HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\ComputerDescriptions"/>
+    <action command="winreg" path="HKLM\Software\Microsoft\Direct3D\MostRecentApplication" name="Name"/>
+    <action command="winreg" path="HKCU\Software\Microsoft\Direct3D\MostRecentApplication" name="Name"/>
+    <action command="winreg" path="HKLM\Software\Microsoft\DirectDraw\MostRecentApplication"/>
+    <action command="winreg" path="HKCU\Software\Microsoft\DirectInput\MostRecentApplication" name="Id"/>
+    <action command="winreg" path="HKCU\Software\Microsoft\DirectInput\MostRecentApplication" name="Name"/>
```

Comment:
What do Direct3D, DirectDraw, and DirectInput have to do with Windows Explorer?

Reply:
I used winsys.ini as a reference...

![2019-07-17 02_53_48-winsys ini - Editor](https://user-images.githubusercontent.com/11881073/61339509-458fc700-a83e-11e9-8348-a7d11d5bd4c9.jpg)